core/fuzz/fuzz_writer.rs

```diff
@@ -32,11 +32,15 @@ const MAX_DATA_SIZE: usize = 16 * 1024 * 1024;
 #[derive(Debug, Clone)]
 struct FuzzInput {
     actions: Vec<WriteAction>,
+    concurrent: usize,
```

Comment:
How about using `Option<usize>` to give us a 50% chance of testing the non-concurrent write case?